homeassistant/components/volvooncall.py

```diff
@@ -123,6 +126,13 @@ def __init__(self, hass, vin, attribute):
         self._state.entities[self._vin].append(self)
 
     @property
+    def entity_id(self):
```

Comment:
Move this into sensor/volvooncall entity and import the domain from sensor. Now you see why you should not import the DOMAIN as DOMAIN inside a platform from a other component then volvooncall.